name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/TradeRepublicPDFExtractor.java

```diff
@@ -2085,9 +2110,19 @@ private void addAccountStatementTransaction_Format02()
                             t.setDateTime(asDate(v.get("date")));
                             t.setAmount(asAmount(v.get("amount")));
                             t.setCurrencyCode(asCurrencyCode(v.get("currency")));
+                            v.getTransactionContext().put(FAILURE, Messages.MsgErrorTransactionTypeNotSupported);
```

Comment:
Hello @christen90 
Thank you for this pull request.
Unfortunately I don't see a test for this change (FAILURE), why this line is added.
Can you add a test case that makes this change plausible?

Regards
Alex